diagnostic.md

```diff
@@ -3,62 +3,72 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
+<head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# object: http://ogp.me/ns/object# article: http://ogp.me/ns/article# profile: http://ogp.me/ns/profile#">
+    <meta charset="utf-8">
+
+    <div role="main" class="main-content">
+            <div itemscope="" itemtype="http://schema.org/SoftwareSourceCode">
+        <div id="js-repo-pjax-container" data-pjax-container="">
+
+        <div role="main" class="main-content">
+                <div itemscope="" itemtype="http://schema.org/SoftwareSourceCode">
+            <div id="js-repo-pjax-container" data-pjax-container="">
+
+        <div class="pagehead repohead instapaper_ignore readability-menu experiment-repo-nav">
+          <div class="container repohead-details-container">
+
+
 
-<!-- your answer ends here -->
 
-## Question 2
+
+
+
+## Question 2parent
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$('.ol').children()
```

Comment:
close. All you need to do is remove the period from infront of the ol because ol is an html element not a class